### PR TITLE
Modernize TitleCount Playbook kit

### PR DESCRIPTION
**Pull request by darrendc**
Modernize TitleCount Playbook kit
BAB-172

### CODE REVIEW

File: spec/pb_kits/playbook/kits/title_count_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+require_relative "../../../../app/pb_kits/playbook/pb_title_count/title_count"
+
+RSpec.describe Playbook::PbTitleCount::TitleCount do
+  subject { Playbook::PbTitleCount::TitleCount }
+
+  it { is_expected.to define_partial }
+
+  describe "#classname" do
+    it "returns namespaced class name", :aggregate_failures do
+      expect(subject.new({}).classname).to eq "pb_title_kit_count_kit"
```
Comment (KFad11) [inline spec/pb_kits/playbook/kits/title_count_spec.rb]: looks like that first `kit` made it fail CI. Should just be  `"pb_title_count_kit"`, right?

Comment (garettarrowood) [reply spec/pb_kits/playbook/kits/title_count_spec.rb]: Looks like this PR is still labeled `wip`.

Comment (darrendc) [reply spec/pb_kits/playbook/kits/title_count_spec.rb]: Yeah, I'm still not working on it.

File: app/pb_kits/playbook/props/percentage.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def validate(value)
     private
 
       def numeric_in_range?(value)
-        (value.is_a?(Numeric) &&
+        (value.is_a?(::Numeric) &&
```
Comment (garettarrowood) [inline app/pb_kits/playbook/props/percentage.rb]: 👍

File: spec/pb_kits/playbook/props/numeric_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+RSpec.describe Playbook::Props::Numeric do
+  describe "#validate" do
+    it "returns true given a valid input", :aggregate_failures do
+      expect(Playbook::Props::Numeric.new.validate(nil)).to eq true
+      expect(Playbook::Props::Numeric.new.validate(12)).to eq true
+      expect(Playbook::Props::Numeric.new.validate(12.3)).to eq true
+    end
+
+    it "returns false given anything something besides a integer or float", :aggregate_failures do
```
Comment (gillianwenhold) [inline spec/pb_kits/playbook/props/numeric_spec.rb]: I think this was meant to be "anything" or "something"

Comment (darrendc) [reply spec/pb_kits/playbook/props/numeric_spec.rb]: good catch

**Pull request merged**